app.js

```diff
@@ -0,0 +1,16 @@
+const kaholoPluginLibrary = require("kaholo-plugin-library");
+const { execute } = require("./ansible-cli");
+const { prepareRunPlaybookPayload } = require("./helpers");
+
+function runPlaybook(params) {
+  const payload = prepareRunPlaybookPayload(params);
+
+  return execute({
+    command: "ansible-playbook",
+    config: payload,
```

Comment:
considering above, the `execute` function accepts some kind of `params` not a config.